lib/Controller/SettingsController.php

```diff
@@ -197,6 +197,9 @@ public function setAppValue($type, $value) {
 			case SettingsService::SHOW_NOT_GROUPED_KEY:
 				$this->settingsService->setShowNotGrouped($value === 'true' ? true : false);
 				break;
+			case SettingsService::OFUSCATE_FACE_THUMBS_KEY:
```

Comment:
`OFUSCATE_FACE_THUMBS_KEY` -> `O**B**FUSCATE_FACE_THUMBS_KEY`

`getOfuscateFaces` -> `getO**b**fuscateFaces`

Reply:
actually, on a lot of places

Reply:
Ohhh..